Client/Frontend/Strings.swift

```diff
@@ -3716,6 +3716,61 @@ extension String {
             tableName: "Shopping",
             value: "Launch analyzer on Fakespot.com",
             comment: "Text for the analyzer button, from No Analysis Card View displayed in the shopping review quality bottom sheet.")
+        public static let ReviewQualityCardLabelTitle = MZLocalizedString(
+            key: "", // Shopping.ReviewQualityCard.Label.Title.v119
+            tableName: "Shopping",
+            value: "How we determine review quality",
+            comment: "Title of the 'How we determine review quality' card displayed in the shopping review quality bottom sheet.")
+        public static let ReviewQualityCardExpandedAccessibilityLabel = MZLocalizedString(
+            key: "", // / Shopping.ReviewQualityCard.Label.Title.v119
+            tableName: "Shopping",
+            value: "How we determine review quality Card Expanded",
+            comment: "Accessibility label for the down chevron, from How we determine review quality Card View displayed in the shopping review quality bottom sheet.")
```

Comment:
I think it's easier to read the comment if we do as you've done in the first comment from `ReviewQualityCardLabelTitle`. So adding `''` around the name of the card. All others comments I think would benefit from that.
```suggestion
            comment: "Accessibility label for the down chevron, from the 'How we determine review quality' Card View displayed in the shopping review quality bottom sheet.")
```